src/components/utils/WaterSplashes.coffee

```diff
@@ -52,16 +52,17 @@ Crafty.c 'WaterSplashes',
               bottomDesaturation: @bottomDesaturation
               alpha: @waterAlpha
               gravity: 0.2
-              (prev) ->
+              (prev, fd) ->
+                mul = fd.dt / (1000 / Crafty.timer.FPS())
```

Comment:
Make this a bit cleaner, maybe like `fd.dt / 1000` and updating the things multiplied to a value per second